packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,294 @@ components:
             created
           example: 1587667174725
 
+    asset:
+      type: object
+      table: asset
+      additionalProperties: false
+      required:
+        - name
+      properties:
+        id:
+          type: string
+          readOnly: true
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+          index: true
+        type:
+          type: string
+          enum:
+            - video
+            - audio
+          description: Type of the asset.
+          example: video
+        playbackId:
+          type: string
+          example: eaw4nk06ts2d0mzb
+          description: Used to form playback URL and storage folder
+        userId:
+          type: string
+          readOnly: true
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+          index: true
+          description: owner of the asset
+        deleted:
+          type: boolean
+          description: Set to true when the asset is deleted
+        objectStoreId:
+          type: string
+          description: Object store ID where the asset is stored
+          writeOnly: true
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        name:
```

Comment:
Hmm, I think we will want to get the filename somehow on the import asset flow in the end to populate this. Or should it be populated on the request to start the import instead? Maybe both, I'll do it on the import task as well since it's easy but the user can provide one on the request to start the import, which if provided should be used instead of the one that the import task gets.